cscope.py

```diff
@@ -170,10 +187,32 @@ class CscopeVisiter(sublime_plugin.TextCommand):
     def __init__(self, view):
         self.view = view
 
-    def run(self, edit):
+    def run(self, edit, **args):
         if self.view.settings().get('syntax') == CSCOPE_SYNTAX_FILE:
+            if args.get('direction'):
```

Comment:
Can you help me understand what you're doing here? I gather the concept is to allow navigation in the results buffer using direction increments? Is this assuming there's only 1 cscope results window? I frequently have multiple cscope results windows. I'm not sure what the goal is here.

Reply:
Yes, keyboard navigation. I also have multiple cscope results windows, no problems. Saved results and currently selected result are held in settings 'results' and 'cscope_results_sel' per cscope results buffer/view.